RELEASE.md

```diff
@@ -52,4 +55,4 @@ The following steps should be always run, regardless of whether an automated or
    1. Mark this version of Stencil as 'released' in JIRA on the 'Releases' page.
    2. Move the task card in this current sprint to the 'Done' swim-lane.
    3. Stub out the next release and task for the release in JIRA.
-5. Ensure all GitHub Issues associated with stories/tasks that shipped in this version of Stencil are closed.
+5. Ensure all GitHub Issues associated with stories/tasks that shipped in this version of Stencil are closed.
```

Comment:
Can we add that last step about commenting the release version number too?

```suggestion
5. Ensure all GitHub Issues associated with stories/tasks that shipped in this version of Stencil are closed.
    1. For each issue, add a comment stating the version of Stencil that included the fix/feature (be sure to update the version number _and_ tag):
```md
The fix for this issue has been released as a part of today's [Stencil vNUMBER release](https://github.com/ionic-team/stencil/releases/tag/TAG). 
```  
```